src/services/language-services/hover/ui-enum-hover.ts

```diff
@@ -0,0 +1,507 @@
+import * as vscode from 'vscode';
+
+export const ACCURACY = vscode.languages.registerHoverProvider('html', {
```

Comment:
This file also has a lot of copied logic, it can be done in the same way as I've suggested for the layout hovers. In fact both files can share a common method for registering the hover provider.